python/lsst/ctrl/bps/drivers.py

```diff
@@ -72,20 +75,45 @@ def _init_submission_driver(config_file):
         Batch Processing Service configuration.
     """
     config = BpsConfig(config_file, BPS_SEARCH_ORDER)
+
+    # Override config with command-line values
+    # Handle diffs between pipetask argument names vs bps yaml
+    translation = {"input": "inCollection",
+                   "output_run": "outCollection",
+                   "qgraph": "qgraphFile",
+                   "pipeline": "pipelineYaml"}
+    for key, value in kwargs.items():
+        # Don't want to override config with None values
+        if value:
```

Comment:
If your only concern is to not override the config values with `None`s then
```python
if value is not None:
    ...
```
feels like be a better option (bonus, no need for the comment :wink:). If you're also worried about overriding existing values with empty strings then the comment is not entirely accurate.

Reply:
I cannot think of use case at the moment to need command line options to set these particular values as empty strings.  Changing comment to include empty string.